spec/features/authenticated_user_security_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.feature "authenticated user security" do
+  scenario "authenticated users cannot view another user's private data" do
+      # As an authenticated user
```

Comment:
i LOVE how easy Fabricate has made things.  It makes the tests look SO much nicer!